modules/core/exported/commitment.go

```diff
@@ -28,7 +28,6 @@ type Prefix interface {
 // Path implements spec:CommitmentPath.
 // A path is the additional information provided to the verification function.
 type Path interface {
```

Comment:
I had to keep the interface because there was an import cycle between 02-client and 23-commitment, if I tried to use `commitmenttypes.MerklePath` as the type for `path` in the `Verify(Non)Membership` functions of the client state interface.